content/src/fieldConfig/business-formation.json

```diff
@@ -186,9 +186,9 @@
       },
       "canGetDistribution": {
         "label": "Partnership Rights - Receive distributions",
-        "body": "Do limited partners have the right to receive distributions from a partner? (This can  include a return of all or any part of their partner's contributions).",
-        "reviewStepYes": "**Yes**, the limited partners have the right to receive distributions from a partner. (This can  include a return of all or any part of their partner's contributions).",
-        "reviewStepNo": "**No**, the limited partners do not have the right to receive distributions from a partner. (This can  include a return of all or any part of their partner's contributions)."
+        "body": "Do limited partners have the right to receive distributions from a partner? (This can include a return of all or any part of their partner's contributions).",
+        "reviewStepYes": "**Yes**, the limited partners have the right to receive distributions from a partner. (This can include a return of all or any part of their partner's contributions).",
+        "reviewStepNo": "**No**, the limited partners do not have the right to receive distributions from a partner. (This can include a return of all or any part of their partner's contributions)."
```

Comment:
These weren't explicitly breaking tests, but removed the additional spaces as cleanup.